src/OpenXRApi.cpp

```diff
@@ -506,10 +533,10 @@ OpenXRApi::OpenXRApi() {
 	glGenTextures(1, &depthbuffer);
 	glBindTexture(GL_TEXTURE_2D, depthbuffer);
 	glTexImage2D(GL_TEXTURE_2D, 0, GL_DEPTH_COMPONENT24,
-	             configuration_views[0].recommendedImageRectWidth,
-	             configuration_views[0].recommendedImageRectHeight, 0,
-	             GL_DEPTH_COMPONENT, GL_UNSIGNED_INT, 0);
-      */
+		configuration_views[0].recommendedImageRectWidth,
+		configuration_views[0].recommendedImageRectHeight, 0,
+		GL_DEPTH_COMPONENT, GL_UNSIGNED_INT, 0);
+	*/
```

Comment:
Actually this block can be deleted, just a leftover from the example code I copied.